game-core/src/main/java/games/strategy/debug/ErrorConsole.java

```diff
@@ -1,63 +0,0 @@
-package games.strategy.debug;
-
-import games.strategy.triplea.settings.ClientSetting;
-import games.strategy.triplea.ui.ErrorHandler;
-import games.strategy.ui.SwingAction;
-import games.strategy.util.Interruptibles;
-
-/**
- * A debug console window that displays the standard output and standard error streams.
- */
-public final class ErrorConsole extends GenericConsole {
```

Comment:
This was merged up with `GenericConsole`, the only other child of `GenericConsole` was `PerformanceConsole` and that was removed.